README.md

```diff
@@ -31,3 +31,13 @@ $ curl http://localhost:8080/events
 $ curl http://localhost:8081/healthcheck
 {"deadlocks":{"healthy":true},"hibernate":{"healthy":true}}
 ```
+
+## Build Tools
+
+### Checkstyle
+
+Prior to committing, run checkstyle to confirm your changes did not introduce additional checkstyle errors.
```

Comment:
I pulled your branch, and I noticed this goal would fail to execute unless I first did `mvn clean install`. We'll probably need to hook this into the build process, or document it here. I think we're already planning to do that though, so maybe this is moot